src/test/java/internship/tests/MyNewslettersTest.java

```diff
@@ -0,0 +1,33 @@
+package internship.tests;
+
+import internship.framework.core.TestBase;
+import internship.framework.pages.MyAccountPage;
+import internship.framework.pages.MynewslettersPage;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+public class MyNewslettersTest extends TestBase {
+    private final ThreadLocal<SoftAssert> softAssert = new ThreadLocal<>();
+
+    @Test(testName = "My Newsletters Test",
+            description = "Testing My Newsletters functionality")
+    public void TestingMyNewslettersFunctionality() {
+        softAssert.set(new SoftAssert());
+        new LoginPageTest().checkLoginPage();
```

Comment:
it is not a good practice to use other test's method

Reply:
Fixed.